mods/common/languages/en.ftl

```diff
@@ -727,31 +727,31 @@ description-actor-tags-overlay = toggles actor tags overlay.
 
 ## DevCommands
 notification-cheats-disabled = Cheats are disabled.
-notification-invalid-cash-amount = Invalid amount of cash.
+notification-invalid-cash-amount = Invalid cash amount.
 description-toggle-visibility = toggles visibility checks and minimap.
 description-give-cash = gives the default or specified amount of money.
-description-give-cash-all = gives the default or specified amount of money to all players and ai.
+description-give-cash-all = gives the default or specified amount of money to all players and AI.
 description-instant-building = toggles instant building.
 description-build-anywhere = toggles the ability to build anywhere.
 description-unlimited-power = toggles infinite power.
 description-enable-tech = toggles the ability to build everything.
-description-fast-charge = toggles almost instant support power charging.
+description-fast-charge = toggles near-instant support power charging.
 description-dev-cheat-all = toggles all cheats and gives you some cash for your trouble.
 description-dev-crash = crashes the game.
 description-levelup-actor = adds a specified number of levels to the selected actors.
 description-player-experience = adds a specified amount of player experience to the owner(s) of selected actors.
-description-power-outage = causes owner(s) of selected actors to have a 5 second power outage.
+description-power-outage = causes a 5-second power outage for the owner(s) of selected actors.
 description-kill-selected-actors = kills selected actors.
-description-dispose-selected-actors = disposes selected actors.
+description-dispose-selected-actors = disposes of selected actors.
```

Comment:
The original wording is correct - it's a technical action. The actors are being disposed. They are not being disposed of.